mitodl/micromasters
# add course run display to staff view of learner page

#### What are the relevant tickets?

closes #2841 

#### What's this PR do?

this adds a display that shows the learner's past course runs.

#### How should this be manually tested?

You should be able to see it for your own user (on `/learner`) provided you have a `staff` or `instructor` role on a program your user is enrolled in.

Alternatively, you can use this patch: https://github.com/aliceriot/mm_patches/blob/master/mock_api.patch

which should allow you see a whole bunch of different course states.
